### PR TITLE
Use keyscript option from cryptsetup in more generic way

### PR DESCRIPTION
**Pull request by Vincent43**
Currently yubikey-luks register itself as 'keyscript' in a non standard way. It's prone to breakeges when cryptsetup change its internals which happened recently in Ubuntu and Debian. This commit switches to use 'keyscript' in canonical way and adds documentation for it. It also cleans now redundant code.

Fixes https://github.com/cornelinux/yubikey-luks/issues/37

### CODE REVIEW

**Comment by Vincent43** [comment]
Hi @cornelinux 
This PR aims to fix some breakages that occurred in Ubuntu and Debian after cryptsetup changed some of its inernals. See
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=904162#15
https://github.com/cornelinux/yubikey-luks/issues/37
https://github.com/cornelinux/yubikey-luks/pull/35

In the end result it simplifies existing code. Please consider merging.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,23 @@ After changing this file, you need to run
 
 so that the changes get transferred to the initramfs.
 
+Enable yubikey-luks initramfs module
+-------------------------
+
+In order to use yubikey-luks for unlocking LUKS encrypted volume at boot you may append keyscript=/usr/share/yubikey-luks/ykluks-keyscript to the /etc/crypttab file. Example:
```
Comment (cornelinux) [inline README.md]: "may" is a bit week here.
I would recommend "must".
Down below is the alternative.

However, I realize, that this is a bit tricky - when I was testing this, I actually missed this point and was wondering, why it did not work out.
I think we somehow need to either add this to the postinst of the debian package or we should output a hint, after running ``yubikey-luks-enroll``.

I also see, that the text from README.md is actually missing in the debian package.
We should put it in the man page. What do you think?

**Comment by cornelinux** [comment]
So dropping script-top and script-bottom make #35 obsolete?

File: README.md (hunk 1 above)
Comment (Vincent43) [reply README.md]: We can put it in manpage (which would need to rewrite it for man format) or install as is in `/usr/share/doc/yubikey-luks`.

**Comment by Vincent43** [comment]
> So dropping script-top and script-bottom make #35 obsolete?

Yes, my PR solves the same problem but in a way that was recommended by debian cryptsetup maintainer. Also #35 would break compatibility on older debian/ubuntu systems so it would need some tweaking.

**Pull request merged**